app/helpers/ci_status_helper.rb

```diff
@@ -51,4 +51,8 @@ def render_ci_status(ci_commit)
       ci_status_icon(ci_commit)
     end
   end
+
+  def ci_widget_summary(commit)
+    "#{commit.statuses_with_matching_status} of #{commit.latest_generic_statuses.length + 1} checks #{commit.human_status}"
```

Comment:
The `latest_ci_statuses` are always grouped into one 'check' and the whole block is show shown if `@merge_request.ci_commit` is true, I assumed this means that there is at least one CI build.